src/components/quote/_macro.njk

```diff
@@ -0,0 +1,6 @@
+{% macro onsQuote(params) %}
+    <blockquote class="quote">
+        <span class="quote__text">{{ params.text | safe }}</span>
```

Comment:
Could be good to set this up so it can be utilised using [call](https://mozilla.github.io/nunjucks/templating.html#call)

```suggestion
        <span class="quote__text">{{ caller() if caller else params.text | safe }}</span>
```